src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -0,0 +1,383 @@
+        + " response.";
+    public static final String TABLE = "| | |\r\n"
+        + "|--|--|\r\n"
+        + "|Application date | %s\r\n"
+        + "|Details of the application | %s\r\n"
+        + "Application file upload | %s";
+    public static final String GROUP_B = "You do not need to respond to this application.<br>";
+    public static final List<String> GROUP_B_TYPES = List.of("Change my personal details", "Consider a decision "
+        + "afresh", "Reconsider a judgment", "Withdraw my claim");
+    public static final String OPEN = "Open";
+    public static final String CLOSED = "Closed";
+
+    private static final String REPLY_OUTPUT_NAME = "%s Reply.pdf";
+    private static final String REPLY_TEMPLATE_NAME = "EM-TRB-EGW-ENG-01212.docx";
+
+    private static final String RESPONDENT_REPLY_MARKUP_FOR_REPLY = "|Response %s | |\r\n"
+        + "|--|--|\r\n"
+        + "|Response from | %s|\r\n"
+        + "|Response date | %s|\r\n"
+        + "|What’s your response to the %s’s application? | %s|\r\n"
+        + "|Supporting material | %s|\r\n"
+        + "|Do you want to copy this correspondence to the other party to satisfy the Rules of Procedure? | %s|\r\n"
+        + "\r\n";
+    private static final String RESPONDENT_REPLY_MARKUP_FOR_DECISION = "|Response %s | |\r\n"
+        + "|--|--|\r\n"
+        + "|Response from | %s|\r\n"
+        + "|Response date | %s|\r\n"
+        + "|Details | %s|\r\n"
+        + "|Supporting material | %s|\r\n"
+        + "\r\n";
+    private static final String ADMIN_REPLY_MARKUP = "|Response %s | |\r\n"
+        + "|--|--|\r\n"
+        + "|Response | %s|\r\n"
+        + "|Date | %s|\r\n"
+        + "|Sent by | %s|\r\n"
+        + "|Case management order or request? | %s|\r\n"
+        + "|Response due | %s|\r\n"
+        + "|Party or parties to respond | %s|\r\n"
+        + "|Additional information | %s|\r\n"
+        + "|Description | %s|\r\n"
+        + "|Document | %s|\r\n"
+        + "%s"
+        + "|Name | %s|\r\n"
+        + "|Sent to | %s|\r\n"
+        + "\r\n";
+    private static final String ADMIN_REPLY_MARKUP_MADE_BY = "|%s made by | %s|\r\n";
+    private static final String IS_CMO_OR_REQUEST_CMO = "Case management order";
+    private static final String IS_CMO_OR_REQUEST_REQUEST = "Request";
+    private static final String COPY_TO_OTHER_PARTY_YES = "I confirm I want to copy";
+    private static final String COPY_TO_OTHER_PARTY_NO = "I do not want to copy";
+
+    private TseHelper() {
+        // Access through static methods
+    }
+
+    /**
+     * Create fields for application dropdown selector.
+     * @param caseData contains all the case data
+     */
+    public static DynamicFixedListType populateSelectApplicationDropdown(CaseData caseData) {
+        if (CollectionUtils.isEmpty(caseData.getGenericTseApplicationCollection())) {
+            return null;
```

Comment:
again returning null? could we not return an error to the user to say 'No applications' or something like this?